src/borg/repository.py

```diff
@@ -1084,6 +1093,9 @@ class LoggedIO:
     class SegmentFull(Exception):
         """raised when a segment is full, before opening next"""
 
+    class AtticSegment(Error):
+        """Attic segment detected [segment {}]"""
```

Comment:
the other magic related message also specifically mentions offset 0.
this is to be consistent with other error messages always giving segment and offset.